chia/consensus/block_body_validation.py

```diff
@@ -125,12 +124,12 @@ async def validate_block_body(
     get_coin_records: Callable[[Collection[bytes32]], Awaitable[List[CoinRecord]]],
     block: Union[FullBlock, UnfinishedBlock],
     height: uint32,
-    npc_result: Optional[NPCResult],
+    conds: Optional[SpendBundleConditions],
```

Comment:
the documentation of this function, below, should be updated as well. and it should say that this must be set for transaction blocks and must be `None` for non-transaction blocks